.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Please have a look at the following list and replace the "[ ]" with a "[x]" if
+the answer to this question is yes.
+
+- [ ] For new contributors: Did you add yourself to the "Authors.rst" file?
+- [ ] For bugfixes and new functionality: Did you add an entry to the
+      "Changes.rst" file?
+- [ ] For bugfixes: Did you add a regression test?
```

Comment:
Can these be grouped together as in:

For bugfix pull requests:
   item1
   item2
   ...

For new functionality: